ibm/mas_devops/roles/suite_app_config/README.md

```diff
@@ -97,7 +132,7 @@ Example Playbook
 
     mas_app_ws_spec:
       bindings:
-        jdbc: "{{ mas_appws_jdbc_binding | default( 'system' , true) }}"
+        jdbc: "{{ mas_appws_jdbc_binding | default( 'workspace-application' , true) }}"
```

Comment:
Why change the default? Can we have system (the most embracing scope) and set workspace-application as an exception case via environment variable? Any particular reason to use `workspace-application` scope and not `system`?

Reply:
Well that's just because Manage's database is configured by default at workspace level for Manage (you setup Manage database not during Manage operator deployment page, but during Manage activation page...), so just thought about having it consistent with what the playbooks should do? I don't think there's any different tbh, If you see any problems with that I can remove it and for IVT envs i just set this via env var... i mean the ansible var with `mas_appws` prefix is really something that correlates to `workspace` right? So this is really just about consistency.

Reply:
Keep default as system .. we don't want to worry about the knock on effects of changing this really.  It just complicates getting this merged in.  In general the default mode in this collection is "deploy for the smallest footprint", which would mean sharing databases.